setup.py

```diff
@@ -12,12 +12,33 @@
 needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 
+# Check Python version
+if sys.version_info[:2] < (3, 6):
+    print('This package requires a verion of Python>=3.6.'
+          'Please upgrading your version of Python to use MembraneCurvature.')
+    sys.exit(-1)
+
+# Add release
+RELEASE = "0.0.0-dev0"
+
+# Check numpy is installed
 try:
+    import numpy as np
+except ImportError:
+    print("MembraneCurvature requires numpy>=1.20.2")
```

Comment:
I would not include the version requirement here, just keep information in _one_ place (DRY = don't repeat yourself). It's enough to fail here if you don't have numpy. Versions are handled by the package manager.